src/app/components/CurrencyInputPanel/index.tsx

```diff
@@ -123,6 +126,8 @@ export default function CurrencyInputPanel({
   customBalanceText,
   bg = 'bg2',
   placeholder = '0',
+  className,
+  balanceList,
```

Comment:
I'd like to make `balanceList` data type as   `[key: string]: BigNumber;`

Reply:
I will refactor the code. after that,  please review @Chunn

Reply:
In the future, the bignumber data type will be changed with `CurrencyAmount` data type.